docs/blog/2019-08-06-gazette-august/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Gatsby Gazette: The August Edition"
+date: 2019-08-06
+author: Sidhartha Chatterjee
+excerpt: "Welcome to the new and improved Gatsby Gazette"
+tags:
+  - gazette
+---
+
+Remember when I said I'll see you folks next week with a jam packed Gazette? It's been 3 months! 😱 If it's any consolation though, this one _is_ a jam packed edition.
+
+![Meme joking about how I said I'll be back in a week and readers are now skeletons because 3 months passed](./meme.jpg)
+
+Welcome to the _new_ and _improved_ Gatsby Gazette. It's been a while. We have a lot to talk about.
+
+This edition features our work on making Gatsby more stable, the Themes Jam, some solid documentation improvements, exciting livestreams featuring Jason _Blitz_ Lengstorf, highlights from the community and the revamped Core Maintainers meeting.
+
+## Team Highlights
+
+### Snow Leopard Release
+
+The past few weeks has been all about what we're calling the Snow Leopard release (kudos to [Dustin](https://twitter.com/schaudustin) for the name).
```

Comment:
I don't really deserve any kudos :) Not a me original ™️

Reply:
Hard disagree. You deserve credit for naming this _and_ the idea which I'll add now.